packages/cojson/src/localNode.ts

```diff
@@ -419,11 +424,12 @@ export class LocalNode {
   subscribe<T extends RawCoValue>(
     id: CoID<T>,
     callback: (update: T | "unavailable") => void,
+    skipRetry?: boolean,
   ): () => void {
     let stopped = false;
     let unsubscribe!: () => void;
 
-    this.load(id)
+    this.load(id, skipRetry)
```

Comment:
This change should be covered with a test in `cojson` to have more visibility on it's effect.

Check the other retry-related tests on the `sync.load.test.ts` file